packages/core/src/codecs/base64-codec.ts

```diff
@@ -0,0 +1,44 @@
+/********************************************************************************
+ * Copyright (c) 2018 - 2019 Contributors to the Eclipse Foundation
```

Comment:
2020 instead of 2019

It is probably needed in other files

Reply:
updated all copyright years affected by this PR